### DISCUSSION TITLE
How to model multi-scale RVE : some melt pools in which some grains are included.

### DISCUSSION THREAD

**Discussion by mikejwg** [neperfepx/neper#634] [Q&A] [answered]
HI, anybody,

Can we model the following multiscale RVE using Neper Multiscale tessellations?
Any suggestions are very appreicated.

The neighbor melt pools are connected to each other by cohesive surfaces. The melt pool that is created because of laser scanning in SLM is shown and b), shows the borders of melt pools connected to each other (dashed lines) where each of these pools contains several grains inside, which are connected to each other through grain boundaries.

Multiscale Tessellations(https://neper.info/generation.html?highlight=multiscale#multiscale-tessellations)
![1-s2](https://user-images.githubusercontent.com/57177281/209808592-1ad71917-2ea3-4e53-94a9-1b47886529f7.jpg)

**Comment by rquey** [comment, marked as answer]
This is not straightforward to do in Neper, primarily because the melt pools are non-convex.

Where does the figure come from?

**Reply by mikejwg** [reply]
https://doi.org/10.1016/j.msea.2018.11.125
